src/Native/Runtime/thread.cpp

```diff
@@ -398,7 +387,9 @@ void Thread::Destroy()
     }
 #endif // !PROJECTN
 
-    RedhawkGCInterface::ReleaseAllocContext(GetAllocContext());
+    gc_alloc_context* m_alloc_context = GetAllocContext();
+    dead_threads_non_alloc_bytes += m_alloc_context->alloc_limit - m_alloc_context->alloc_ptr;
```

Comment:
I would be better to keep the dead_threads_non_alloc_bytes counter in one of the gc*.* files (gcrhenv.cpp/gchelpers.cpp).

Reply:
Since you've marked the tracking code, did you mean the code or the variable?

Reply:
Both. Basically, keep just the call to `RedhawkGCInterface::ReleaseAllocContext(GetAllocContext());` in this file.